modules/APIEngine.py

```diff
@@ -9,104 +9,122 @@
 import json
 import datetime
 import os
-from modules.ConfigEngine import get_config
+import abc
 
-dateSelected = datetime.datetime.now().strftime("%Y-%m-%d")  # Date du jour pour parse le JSON
+from ConfigEngine import get_config
+
+dateSelected = datetime.datetime.now().strftime(
+    "%Y-%m-%d")  # Date du jour pour parse le JSON
 directory = os.path.join(os.path.dirname(__file__), '../')
 
 
 # dateSelected = '2020-04-29' #Permet de sélectionner une date 'manuellement'
 
-def get_data(source):
-    if source == "WORLDOMETERS":
-        worldometers_api = requests.get("https://coronavirus-19-api.herokuapp.com/countries/France")
-        worldometers_data = worldometers_api.json()
+class DataRequester(abc.ABC):
+    def __init__(self):
+        super().__init__()
 
-        '''
-		print("\nDonnées de WORLDOMETERS:")
-		print(WorldometersData)
-		print("\n")
-		'''
+    @abc.abstractclassmethod
+    def get_data(self):
+        pass
 
-        with open(directory + 'data/todayWorldometersData.json', 'w') as data:
-            json.dump(worldometers_data, data)
 
-        return worldometers_data
```

Comment:
This is very bad because your function are to big and it has more than one responsibility. That's why I replace this by two classes.